src/jit/flowgraph.cpp

```diff
@@ -21742,58 +21742,76 @@ void Compiler::fgDebugCheckBlockLinks()
     }
 }
 
+// UniquenessCheckWalker keeps data that is neccesary to check
+// that each tree has it is own unique id and they do not repeat.
+class UniquenessCheckWalker
```

Comment:
I don't know what others think but I would have preferred a local class without all the standard documentation clutter. This looks like overkill just to set a bit in a bitvector. Maybe that's just me :)

Reply:
I do not like to write them, but when they are already written I am fine with them)
I should ask @BruceForstall's opinion when he comes back from his vacation.